src/style.css

```diff
@@ -0,0 +1,113 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf) ;
+  font-weight: normal;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+}
+
+body {
+  margin-top: 0;
+}
+
+.wraper-big {
+  margin-bottom: 20px;
+  position: relative;
+  margin-top: 20px;
+}
+
+.wraper-small {
+  position: relative;
+}
+
+input[type="search"]::-webkit-search-cancel-button {
+  opacity: 0;
+}
+
+.icon-big {
+  color: #687c94;
+  position: absolute;
+  width: 19px;
+  bottom: 26px;
+  left: 26px;
+}
+
+.icon-small {
+  position: absolute;
+  color: #687c94;
+  width: 11px;
+  bottom: 16px;
+  left: 13px;
+}
+
+.big {
+  box-sizing: border-box;
+  height: 70px;
+  width: 100%;
+  font-family: Avenir, sans-serif;
+  padding-left: 62px;
+  line-height: 22px;
+  font-size: 16px;
+  color: #3d4e61;
+  background-color: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.small {
+  height: 42px;
+  box-sizing: border-box;
+  width: 100%;
+  font-family: Avenir, sans-serif;
+  padding-left: 33px;
+  line-height: 19px;
+  font-size: 14px;
+  color: #3d4e61;
+  background-color: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.big:focus {
+  font-family: Avenir, sans-serif;
+  font-weight: bold;
+  background-color: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border-radius: 3px;
+  outline: none;
+}
+
+.small:focus {
+  font-family: Avenir, sans-serif;
+  font-weight: bold;
+  background-color: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border-radius: 3px;
+  outline: none;
+}
+
+.small:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.big:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.big::placeholder {
+  font-family: Avenir, sans-serif;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 22px;
+}
+
+.small::placeholder {
+  font-family: Avenir, sans-serif;
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 19px;
+}
```

Comment:
You duplicate a bunch of styles. [Checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md) the 2nd point

Reply:
I was really upset when try to write code css, and one aim was make solution. And when it was completed i click pull requst and all. in future i be better